targets/targets.json

```diff
@@ -5043,8 +5043,13 @@
             "SERIAL",
             "SLEEP",
             "SPI",
-            "SPISLAVE"
-        ]
+            "SPISLAVE",
+            "USTICKER"
+        ],
+        "overrides": {
+            "tickless-from-us-ticker"   : true,
+            "boot-stack-size"           : "0x400"
```

Comment:
Why do we also change `boot-stack-size` in this PR which addresses a baremetal link error?
I am assuming that there is not enough memory on this target to accommodate the default 4K stack size?

Reply:
https://github.com/ARMmbed/mbed-os/pull/12155#issuecomment-570980438
https://github.com/ARMmbed/mbed-os/pull/12134#discussion_r366242360

Reply:
I don't know why we also need to change the boot stack size nor which value it should be set to: 0x800 or 0x400:
https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_NORDIC/TARGET_MCU_NRF51822/device/TOOLCHAIN_GCC_ARM/TARGET_MCU_NORDIC_32K/NRF51822.ld/#L4

https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_NORDIC/TARGET_MCU_NRF51822/device/TOOLCHAIN_IAR/TARGET_MCU_NORDIC_32K/nRF51822_QFAA.icf/#L7

@mprse You worked on this some time ago, can you give your view?